webapp/src/ts/modules/tasks/tasks.component.ts

```diff
@@ -161,9 +161,7 @@ export class TasksComponent implements OnInit, OnDestroy {
         this.tasksActions.setTasksLoaded(true);
       }
 
-      telemetryData.end = Date.now();
-      const telemetryEntryName = !this.tasksLoaded ? `tasks:load` : `tasks:refresh`;
-      this.telemetryService.record(telemetryEntryName, telemetryData.end - telemetryData.start);
+      track?.stop(recordApdex, false);
```

Comment:
@garethbowen, quick question: how important is keeping the old telemetry names? 
We intend to prefix performance records with `perf:<component>:<etc>`, but thinking that we need backward compatibility when adding the ApDex, I added a flag to skip the prefix. 
It's a small thing, but having the name convention across would be nicer and avoid this extra check.

Reply:
I think we have to keep backwards compatibility here - it's part of the API and someone may be using it.